docs/report_generation.adoc

```diff
@@ -193,3 +193,24 @@ test.finalizedBy jgivenTestReport
 
 For additional information about the Gradle plugin refer to
 https://plugins.gradle.org/plugin/com.tngtech.jgiven.gradle-plugin
+
+=== Configuration File
+
+JGiven will optionally load a configuration properties file, defaulting to:
+`jgiven.properties`. The path to the configuration can be customized with the system property:
+----
+jgiven.config.path
+----
+The encoding for the file is assumed to be `UTF-8`, but can be customized with the system property:
+----
+jgiven.config.charset
+----
+The following can be defined the properties file:
+----
+jgiven.report.enabled
```

Comment:
we should record the defaults

Reply:
already done in a previous section

Reply:
Nevertheless, it might be nice to have all defaults recalled here again to avoid search throughout the file?